src/AppRoutes.jsx

```diff
@@ -10,6 +10,7 @@ import Header from "./components/Header";
 import Footer from "./components/Footer";
 import AnnouncerController from "./components/Announcer/AnnouncerController";
 import Announcer from "./components/Announcer/Announcer";
+import AppMethodsContext from "./states/legacy/AppMethodsContext";
```

Comment:
Thanks for pointing out, this must be unintentionally added back during reordering commits. It wasn't prominent in my editor that because it thinks .jsx was a compiled code from the corresponding .tsx file. Now I removed AppRoutes.jsx